packages/lesswrong/components/admin/CurationPage.tsx

```diff
@@ -37,6 +37,17 @@ export const CurationPage = ({classes}: {
     }
   });
 
+  const { results: curatedPosts } = useMulti({
```

Comment:
Did you test this in dev to check whether curating a post actually causes this to update ~immediately (as opposed to either requiring a round trip to the server, or not happening at all until a manual refresh)?